### PR TITLE
Pasternak 175 accounts model

### PR DESCRIPTION
**Pull request by MarinaPasternak**
add cookie and acctions for registretion/log out/log in/ get lists
image

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -33,9 +33,20 @@ module.exports = {
     'import/extensions': ['error', 'ignorePackages', {
       js: 'always',
     }],
+    'react/button-has-type': 'off',
     'object-curly-spacing': [2, 'always'],
     'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
     'react/jsx-one-expression-per-line': 0,
+<<<<<<< HEAD
```
Comment (PavelKundenko) [inline .eslintrc.js:40]: Resolve conflicts

File: src/features/add-lesson/add-lesson.js [hunk 2]
```diff
@@ -81,6 +81,35 @@ export const AddLesson = () => {
...
+            </table>
+          </div>
+        </form>
+      </div>
+      <div className={styles.placement}>
+        <button form="form" type="button" className="btn btn-secondary mx-5 btn-lg">Cancel</button>
+        <button form="form" type="button" className="btn btn-success btn-lg">Save</button>
+      </div>
+    </div>
+  );
```
Comment (PavelKundenko) [inline src/features/add-lesson/add-lesson.js:114]: It seems to me that this code repeats

File: src/features/add-lesson/add-lesson.scss [hunk 3]
```diff
@@ -15,11 +15,17 @@ h3{
...
+.size {
+  width: 100%;
+}
+=======
 
 .size {
   width: 100%;
 }
 
+>>>>>>> dev
```
Comment (PavelKundenko) [inline src/features/add-lesson/add-lesson.scss:28]: Resolve conflicts

File: src/models/account/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export { authReducer } from './reducer.js';
+export { authWatcher, login, registretion, logOut, fetchUsersList, fetchUnAssignedUserList } from './sagas.js';
```
Comment (PavelKundenko) [inline src/models/account/index.js:2]: Do we really need to export all these sagas? I think that we should export only main watcher

File: src/models/index.js [hunk 5]
```diff
@@ -2,3 +2,7 @@ export { ApiService } from './api-service/index.js';
 export {
   mentorsReducer, loadMentors, loadMentorsWatcher, mentorsIsLoadingSelector, mentorsDataSelector,
 } from './mentors/index.js';
+export {
+  authReducer, authWatcher, login, logOut, registretion, fetchUsersList, fetchUnAssignedUserList,
+  currentUserSelector, newUserSelector, fetchAssignedUsersSelector, fetchUnAssignedUsersSelector, fuserLoading
+} from './account/index.js';
```
Comment (PavelKundenko) [inline src/models/index.js:8]: You can use `export * from './account/index.js';

**Review by PavelKundenko — CHANGES_REQUESTED** [review]

File: src/models/account/index.js (hunk 4 above)
Comment (MarinaPasternak) [reply src/models/account/index.js:2]: Yes, I have errors if I export only authWatcher

**Review by PavelKundenko — APPROVED** [review]

**Review by Icemandov — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**